.github/workflows/build-push-docker-images.yaml

```diff
@@ -0,0 +1,183 @@
+name: Build-Push-Docker-Image
+
+on:
+  push:
+    branches:
+      - build-push-docker-images
+
+jobs:
+  build-push-base-rvt-image:
+
+    name: Build and push RVT base Docker image
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        include:
+          - username: rvt
+            user_uid: 1000
+            user_gid: 1000
+            ubuntu_version: 20.04
+
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push rvt_base
+        uses: docker/build-push-action@v2.2.2
+        with:
+          context: docker/base
+          push: true
+          build-args: |
+            USERNAME=${{ matrix.username }}
+            USER_UID=${{ matrix.user_uid }}
+            USER_GID=${{ matrix.user_gid }}
+            UBUNTU_VERSION=${{ matrix.ubuntu_version }}
+          cache-from: type=registry,ref=thierrymarianne/contrib-rvt_base:latest
+          cache-to: type=inline
+          tags: |
+            thierrymarianne/contrib-rvt_base:latest
```

Comment:
reminder to change this to change this to a repo that is controlled by the rvt team (DOCKERHUB_USERNAME and DOCKERHUB_TOKEN will need to be provided as github secrets in order for this to build once merged upstream)